.github/templates/build.lib.yml

```diff
@@ -43,17 +43,23 @@ deploy-cluster:
     AtlasPrivateKey: #@ secret_AtlasPrivateKey
 #@ end
 
-#@ def cleanupBaas(dependencies = []):
-#@ needs = []
+#@ def cleanupBaas(dependencies = [], targets = []):
```

Comment:
Can we unify the naming here so that both dependencies and targets are named the same way?

Reply:
The "problem" here is that MacOS has one test file (test-macos.yml), but two different tests (xamarin and catalyst), so there's not a 1-to-1 correspondence between dependencies and targets. 
The thing we could do is to create a function that gets the targets given the file dependency, but I'm not sure how much of an advantage we'll get. I agree that now it's kinda ugly

Reply:
Yeah, we can keep it as is, maybe just add a note for why this is needed so that we don't forget.